src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -1,8 +1,158 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" >
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
+
     <l:layout permission="${app.ADMINISTER}" title="${%title}" norefresh="true" type="one-column">
         <l:main-panel>
+            <link rel="stylesheet" type="text/css" href="${resURL}/plugin/atlassian-jira-software-cloud/config.css"/>
+            <script src="${resURL}/plugin/atlassian-jira-software-cloud/config.js"/>
+            <script src="${resURL}/plugin/atlassian-jira-software-cloud/regexTester.js"/>
```

Comment:
resURL is the entire jenkins plugins library, it does some things but basically points to our webapp directory.... this was really hard to figure out :shakes-fist: